cms/djangoapps/contentstore/proctoring.py

```diff
@@ -0,0 +1,127 @@
+"""
+Code related to the handling of Proctored Exams in Studio
+"""
+
+import logging
+
+from django.conf import settings
+
+from xmodule.modulestore.django import modulestore
+
+from edx_proctoring.api import (
+    get_exam_by_content_id,
+    update_exam,
+    create_exam,
+    get_all_exams_for_course,
+)
+from edx_proctoring.exceptions import (
+    ProctoredExamNotFoundException
+)
+
+log = logging.getLogger(__name__)
+
+
+def _is_in_course_tree(item):
```

Comment:
This seems like a useful helper that I'm surprised hasn't been built already (I did a quick search and couldn't find one). Can you move it to ``contentstore/views/helpers.py``.